app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="inaka.lewisdemo"
+    android:versionCode="@android:integer/config_longAnimTime">
```

Comment:
According to **Lewis**:
> **Id**:IllegalResourceRef **Severity**:Warning **Category**:Correctness**Message**: The `android:versionCode` cannot be a resource url, it must be a literal integer.
 **Explanation**: For the `versionCode` attribute, you have to specify an actual integer literal; you cannot use an indirection with a `@dimen/name` resource. Similarly, the `versionName` attribute should be an actual string, not a string resource url.